docs/learn-phragmen.md

```diff
@@ -610,24 +611,26 @@ an optimal manner.
 
 There are several further restrictions put in place to limit the complexity of the election and
 payout. As already mentioned, any given nominator can only select up to 16 validators to nominate.
-Conversely, a single validator can have only {{ polkadot_max_nominators }} nominators. A drawback to
-this is that it is possible, if the number of nominators is very high or the number of validators is
-very low, that all available validators may be "oversubscribed" and unable to accept more
-nominations. In this case, one may need a larger amount of stake to participate in staking, since
-nominations are priority-ranked in terms of amount of stake.
+Conversely, a single validator can have only 128 nominators. A drawback to this is that it is
```

Comment:
Why the change from a dynamic value?

Reply:
It was not rendering properly on the webpage, which I should have fixed instead of just hardcoding it. Any clue how to set it?
<img width="1319" alt="Screen Shot 2021-04-26 at 9 57 16 AM" src="https://user-images.githubusercontent.com/8061957/116105318-0d015c00-a666-11eb-8a60-5e8e6fba69ec.png">

Reply:
interesting enough, I see this rendered properly on wiki.polkadot.network, but not localhost. Do I need to know how to reproduce locally? Where can I get a list of possible variables?